avatica-shaded/dependency-reduced-pom.xml

```diff
@@ -0,0 +1,91 @@
+    <version>10.0.0-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>kafka-connect-storage-common-avatica-shaded</artifactId>
+  <name>kafka-connect-storage-common-avatica-shaded</name>
+  <description>avatica shaded package to replace jackson dependencies without CVEs</description>
+  <inceptionYear>2020</inceptionYear>
+  <build>
+    <plugins>
+      <plugin>
+        <artifactId>maven-shade-plugin</artifactId>
+        <version>3.2.4</version>
+        <executions>
+          <execution>
+            <phase>package</phase>
+            <goals>
+              <goal>shade</goal>
+            </goals>
+            <configuration>
+              <artifactSet>
+                <includes>
+                  <include>org.apache.calcite.avatica:*</include>
+                  <include>com.fasterxml.jackson.core:*</include>
+                </includes>
+              </artifactSet>
+              <filters>
+                <filter>
+                  <artifact>org.apache.calcite.avatica:avatica</artifact>
+                  <includes>
+                    <include>**</include>
+                  </includes>
+                  <excludes>
+                    <exclude>META-INF/MANIFEST.MF</exclude>
+                    <exclude>META-INF/LICENSE</exclude>
+                    <exclude>META-INF/NOTICE</exclude>
+                    <exclude>META-INF/services/com.fasterxml.jackson.core*</exclude>
+                    <exclude>META-INF/maven/com.fasterxml.jackson.core/**</exclude>
+                    <exclude>com/fasterxml/jackson/**</exclude>
+                  </excludes>
+                </filter>
+                <filter>
+                  <artifact>com.fasterxml.jackson.core:*</artifact>
+                  <includes>
+                    <include>**</include>
+                  </includes>
+                </filter>
+              </filters>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+    </plugins>
+  </build>
+  <dependencies>
+    <dependency>
+      <groupId>org.apache.kafka</groupId>
+      <artifactId>connect-api</artifactId>
+      <version>6.0.0-ccs</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>jline</groupId>
+      <artifactId>jline</artifactId>
+      <version>2.12.1</version>
+      <scope>compile</scope>
+    </dependency>
+    <dependency>
+      <groupId>io.confluent</groupId>
+      <artifactId>common-utils</artifactId>
+      <version>6.0.0</version>
+      <scope>compile</scope>
+    </dependency>
+    <dependency>
+      <groupId>io.confluent</groupId>
+      <artifactId>assembly-plugin-boilerplate</artifactId>
+      <version>6.0.0</version>
+      <type>zip</type>
+      <classifier>resources</classifier>
+      <scope>provided</scope>
+    </dependency>
+  </dependencies>
+  <properties>
+    <avatica.version>1.8.0</avatica.version>
+    <jackson.version>2.10.5</jackson.version>
+  </properties>

```

Comment:
Seems strange to have these at the bottom.

Reply:
This files was added unintentionally. It's produced in every run. I thought of adding it to `.gitignore`